test/integration/globals/app-nav.js

```diff
@@ -334,6 +367,11 @@ context('App Nav', function() {
   });
 
   specify('add patient failure', function() {
+    // Flake Warning:
```

Comment:
```
const testDate = new Date(2019, 1, 1).getTime(); // Jan 1, 2019 timestamp

cy.clock(testDate);
// test date relative to testDate
cy.clock().invoke('restore');
```

Reply:
I ended up needed to go with `cy.clock(testDate, ['Date'])` otherwise the app wouldn't start up in the test.